spec/system/provider_interface/see_individual_application_spec.rb

```diff
@@ -44,6 +45,11 @@ def and_the_safeguarding_declaration_feature_flag_is_active
     FeatureFlag.activate('provider_view_safeguarding')
   end
 
+  def and_i_am_permitted_to_see_safeguarding_information
+    ProviderUser.find_by(dfe_sign_in_uid: 'DFE_SIGN_IN_UID')
+      .provider_permissions.update_all(view_safeguarding_information: true)
+  end
+
```

Comment:
@steventux I just accidentally broke this feature while rebasing, but the tests still pass. Can you add a test that the provider user _can't_ see the safeguarding data without the correct permission?

Reply:
https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2097